driver/ppm_events.c

```diff
@@ -1768,19 +1736,7 @@ int f_sys_autofill(struct event_filler_arguments *args)
 
 	for (j = 0; j < evinfo->n_autofill_args; j++) {
 		if (evinfo->autofill_args[j].id >= 0) {
-#ifdef _HAS_SOCKETCALL
-			if (args->is_socketcall && evinfo->paramtype == APT_SOCK) {
```

Comment:
We lose `evinfo->paramtype == APT_SOCK` here; i am not sure this is relevant though.